Batteries/Tactic/Where.lean

```diff
@@ -74,7 +84,8 @@ elab "#where" : command => do
     msg := msg.push <| "universe " ++ MessageData.joinSep levels " "
   -- Variables
   if !scope.varDecls.isEmpty then
-    msg := msg.push <| ← `(command| variable $scope.varDecls*)
+    let vars := scope.varDecls.map (⟨clearTrailing ·⟩)
```

Comment:
The function `Lean.Syntax.unsetTrailing` seems to be doing what you want here, but I haven't tested it.
```suggestion
    let vars := scope.varDecls.map (⟨·.unsetTrailing⟩)
```

Reply:
Ah, great!  I thought that I needed to preserve a space between the various syntax nodes, but it seems like that is already handles somewhere else.  Thanks!